app/index.js

```diff
@@ -48,20 +48,25 @@ const url = 'file://' + resolve(
 console.log('electron will open', url);
 
 app.on('ready', () => {
-  function createWindow (fn) {
+  function createWindow (fn, options = {defaults: {position: undefined, size: undefined}}) {
```

Comment:
It's weird to nest them in a `defaults` object. If you're gonna have defaults, just have an option object.

```js
function createWindow (fn, options = {position: undefined, size: undefined}) {
```

Though, I fail to see the point of this default. It's not even used anywhere.